### DISCUSSION TITLE
CLI users deserve more respect

### DISCUSSION THREAD

**Discussion by rmica** [Chia-Network/chia-blockchain#7500] [General]
Quote from the Pooling User Guide: "If you are using the CLI or a 3rd party plotter, hover your mouse over the question mark in the Plot NFT page, and copy the Pool Contract Address".

Really? If you are using the CLI then "hover your mouse" to get the Pool Contract Address? Why not give us the CLI to read it?

Sincerely.

**Comment by alexwwstx** [comment]
chia plotnft show

**Reply by rmica** [reply]
Thanks.
